src/config.cc

```diff
@@ -484,9 +488,6 @@ void Config::initFieldCallback() {
 }
 
 Config::~Config() {
```

Comment:
Could you delete it and add `~Config() = default;` in `config.h`?
We'd better make sure these trivial function in header to let compilers do more optimization like inlining and call elimination.